src/components/NavigationBar/index.js

```diff
@@ -1,40 +1,61 @@
-import { h } from 'preact'
+import { h, Component } from 'preact'
 import { connect } from 'react-redux'
 import classNames from 'classnames'
 import { compose } from '~utils/func'
 import { setNavigationDisabled } from '../ReduxAppWrapper/store/actions/globals'
 import { withFullScreenState } from '../FullScreen'
 import style from './style.css'
-import {isDesktop} from '~utils/index'
-import {localised} from '../../locales'
+import { isDesktop } from '~utils/index'
+import { localised } from '../../locales'
 
-export const withNavigationDisabledState = connect(({ globals: { isNavigationDisabled }}) => ({ isNavigationDisabled }))
+export const withNavigationDisabledState = connect(
+  ({ globals: { isNavigationDisabled } }) => ({ isNavigationDisabled })
+)
 
-export const withNavigationDisableAction = connect(null, dispatch => ({
-  setNavigationDisabled: value => dispatch(setNavigationDisabled(value))
+export const withNavigationDisableAction = connect(null, (dispatch) => ({
+  setNavigationDisabled: (value) => dispatch(setNavigationDisabled(value)),
 }))
 
-const NavigationBar = ({back, translate, disabled, isFullScreen, className}) =>
-  <div className={classNames(className, style.navigation, {
-    [style.fullScreenNav]: isFullScreen
-  })}>
-    <button
-      type="button"
-      aria-label={translate('back')}
-      onClick={back}
-      className={classNames(style.back, {
-        [style.disabled]: disabled,
-        [style.backHoverDesktop]: isDesktop
-      })}
-    >
-        <span className={style.iconBack} />
-        <span className={style.label} aria-hidden="true">
-          {translate('back')}
-        </span>
-    </button>
- </div>
+class NavigationBar extends Component {
 
-export default compose(
-  withFullScreenState,
-  localised
-)(NavigationBar)
+  componentDidUpdate(prevProps) {
+    if (prevProps.className !== this.props.className && !this.props.disabled) {
```

Comment:
[Minor] I'd say to also check if `this.backBtn` exists (that is - if reference has been assigned).

Reply:
With this change, both the NavigationBar component and the PageTitle will attempt to focus on their ref when they get updated, but you can only focus on one element at time. I guess that the element that will be updated last, will have the focus. I think that this might cause issue with the screen title announcement.

Reply:
I've removed the initial focus on `componentDidMount` in PageTitle while keeping the `aria-live="assertive"` as a compromise.

Reply:
PageTitle is shifting the focus on the heading every time is updated too
```
componentDidUpdate(prevProps) {
    const { title, subTitle } = this.props
    if (title !== prevProps.title || subTitle !== prevProps.subTitle) {	   
      this.container.focus()
    }
  }

Reply:
Ah ok. Missed the title comparison there & didn't notice it. But my screenreader was also reading out stuff from other tabs even with restart. Might have got lost in all the announcements.